src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colours.values().length);
```

Comment:
Don’t create several instances of the same class if you can use only one instance for your purposes. In each call of` getRandomColor() ` `new Random()` will be created.